### DISCUSSION TITLE
Hotkey for Superscript and Subscript?

### DISCUSSION THREAD

**Discussion by Frazzer951** [TriliumNext/Trilium#2695] [Q&A]
Is there a hotkey for superscript and subscript? 
In Word you can use `Ctrl + Shift + '+'` for superscript 
and `Ctrl + '+'` for subscript

**Comment by XXXJetfireXXX** [comment]
As a temporary solution, you can use math formulas (`Ctrl + M`) like so:
![image](https://user-images.githubusercontent.com/43646178/157749101-f89491d7-7517-4bce-8038-a9d49b1f2d5e.png)
or:
![image](https://user-images.githubusercontent.com/43646178/157749322-acfcae58-8c10-42f8-b55a-350a1cbdb264.png)
